services/QuillLMS/app/models/activity.rb

```diff
@@ -221,6 +222,19 @@ def is_evidence?
     classification&.key == ActivityClassification::EVIDENCE_KEY
   end
 
+  def child_activity
```

Comment:
Since this method's implementation is coupled to Evidence, would `evidence_child_activity` be more informative?  (This would also be consistent with the methods `update_evidence_title?` and `update_evidence_child_title`)

Reply:
That's a reasonable suggestion, but I actually wanted to leave this in its more generic form in case we later decide to add additional activities with relationships here.  I can imagine us adding new activity types, or modifying old ones, such that this would be a useful function to hook up to.  I'm not married to it, but it seemed worth leaving that possibility open.